test/videojs-http-streaming.test.js

```diff
@@ -2915,6 +2915,50 @@ QUnit.test('configures eme if present on selectedinitialmedia', function(assert)
   }, 'set source eme options');
 });
 
+QUnit.test('integration: configures eme if present on selectedinitialmedia', function(assert) {
```

Comment:
The test above this tests the same thing, but with all the playlist loaders mocked. Should we keep both?